### PR TITLE
[intro.defs] Remove cross-references to main part

### PR DESCRIPTION
**Pull request by jensmaurer**
Partially addresses ISO/CS 016 (C++20 DIS)

Fixes cplusplus/nbballot#397

### CODE REVIEW

**Review by tkoeppe — APPROVED** [review]
Maybe extend the commit message to say "cross-references from the Terms and Definitions to the main part".

File: source/intro.tex [hunk 1]
```diff
@@ -121,9 +121,7 @@
 \indexdefn{access}%
 \definition{access}{defns.access}
 \defncontext{execution-time action}
-read\iref{conv.lval} or
-modify (\ref{expr.ass}, \ref{expr.post.incr}, \ref{expr.pre.incr})
-the value of an object
```
Comment (zygoloid) [inline source/intro.tex:126]: Can we add some text to the note below to retain these useful xrefs? Perhaps add between the first and second sentence:

> Reads of scalar objects are described in \ref{conv.lval} and modifications of scalar objects are describred in \ref{expr.ass}, \ref{expr.post.incr}, and \ref{expr.pre.incr}.

I think it's important to the use of this definition that we know what these four cases are.

File: source/intro.tex [hunk 2]
```diff
@@ -345,8 +343,8 @@
 
 \definition{iostream class templates}{defns.iostream.templates}
 \defncontext{library}
-templates, defined in \ref{input.output},
-that take two template arguments
+templates that take two template arguments
+and are declared in header \libheader{iosfwd}
```
Comment (zygoloid) [inline source/intro.tex:347]: Can we keep the two conditions the other way around? It makes more sense to me to start with all templates in `<iosfwd>` and then filter by "takes two arguments" than to start with the universe of all templates taking two arguments and then filter by "declared in `<iosfwd>`".

File: source/intro.tex (hunk 1 above)
Comment (jensmaurer) [reply source/intro.tex:115]: Fixed.

File: source/intro.tex (hunk 2 above)
Comment (jensmaurer) [reply source/intro.tex]: Fixed.

**Pull request merged**